workflow.rst

```diff
@@ -438,16 +451,61 @@ transition. The value of this option is any valid expression created with the
                         from: draft
                         to:   reviewed
                     publish:
-                        # or "is_anonymous", "is_remember_me", "is_fully_authenticated", "is_granted"
+                        # or "is_anonymous", "is_remember_me", "is_fully_authenticated", "is_granted", "is_valid"
```

Comment:
this issue can be closed
https://github.com/symfony/symfony-docs/issues/8499
or i miss something?

Reply:
👍🏻 from my side.

If Javier is fine too, please add a „Closes #...“ to the PR header, thank you 😊